extensions/amp-story/1.0/amp-story.js

```diff
@@ -2668,10 +2668,14 @@ export class AmpStory extends AMP.BaseElement {
   fetchLocalizationStrings_(languageCode) {
     const localizationService = getLocalizationService(this.element);
 
-    const localizationUrl = `${calculateScriptBaseUrl(
+    const base = calculateScriptBaseUrl(
       this.win.location,
       getMode(this.win).localDev
-    )}/v0/amp-story.${languageCode}.json`;
+    );
+    const rtvPath = getMode(this.win).localDev
```

Comment:
Nit: Can we do `{localDev, rtvVersion} = getMode(this.win)` and use them with just one call to `getMode`?

Reply:
get a lint error. not allowed to alias these getMode calls since they are transformed i think